server/controllers/stock/reports/stock_inventories.report.handlebars

```diff
@@ -28,71 +32,93 @@
-                {{#equal status 'in_stock'}}<span class="label label-success">{{translate 'STOCK.STATUS.IN_STOCK'}}</span>{{/equal}}
-                {{#equal status 'security_reached'}}<span class="label label-warning">{{translate 'STOCK.STATUS.SECURITY'}}</span>{{/equal}}
-                {{#equal status 'minimum_reached'}}<span class="label label-info">{{translate 'STOCK.STATUS.MINIMUM'}}</span>{{/equal}}
-                {{#equal status 'over_maximum'}}<span class="label label-primary">{{translate 'STOCK.STATUS.OVER_MAX'}}</span>{{/equal}}
-              </td>
-              <td class="text-right">{{avg_consumption}}</td>
-              <td class="text-right">{{S_MONTH}}</td>
-              <td class="text-right">{{S_SEC}}</td>
-              <td class="text-right">{{S_MIN}}</td>
-              <td class="text-right">{{S_MAX}}</td>
-              <td class="text-right">
-                {{#gt S_Q 0}}<span>{{S_Q}}</span>{{else}} 0 {{/gt}}
-              </td>
+          {{!-- for each depots which contains inventories --}}
+          {{#each depots as | items name |}}
+
+            <!-- this is the depot group header -->
+            <tr style="border:none">
+              <th style="border:none; border-bottom: solid black 2px;" class="text-uppercase">
+                {{ name }}
+              </th>
+
+              <th colspan="10" style="border:none; border-bottom: solid black 2px;" class="text-right">
+                ({{ items.length }} {{ translate "TABLE.AGGREGATES.RECORDS" }})
+              </th>
             </tr>
-          {{else}}
-            {{> emptyTable columns=9}}
+
+            <!-- these are the items for each group -->
+            {{#each items as | item | }}
+              <tr>
+                <td>{{code}}</td>
+                <td>{{text}}</td>
+                <td class="text-right" style="font-weight: bold;">{{quantity}}</td>
+                <td>{{unit_type}}</td>
+                <td style="font-weight: bold;">
+                  {{#equal status 'sold_out'}}{{translate 'STOCK.STATUS.SOLD_OUT'}}{{/equal}}
+                  {{#equal status 'in_stock'}}{{translate 'STOCK.STATUS.IN_STOCK'}}{{/equal}}
+                  {{#equal status 'security_reached'}}{{translate 'STOCK.STATUS.SECURITY'}}{{/equal}}
+                  {{#equal status 'minimum_reached'}}{{translate 'STOCK.STATUS.MINIMUM'}}{{/equal}}
+                  {{#equal status 'over_maximum'}}{{translate 'STOCK.STATUS.OVER_MAX'}}{{/equal}}
+                </td>
+                <td class="text-right">{{avg_consumption}}</td>
+                <td class="text-right">{{S_MONTH}}</td>
+                <td class="text-right">{{S_SEC}}</td>
+                <td class="text-right">{{S_MIN}}</td>
+                <td class="text-right">{{S_MAX}}</td>
+                <td class="text-right">
+                  {{#gt S_Q 0}}<span>{{S_Q}}</span>{{else}} 0 {{/gt}}
```

Comment:
This is kind of hard to read - can you move this logic into the JS?  That way you can just write:
```html
{{ S_Q }}
```